src/loaders/common/loader_helper.py

```diff
@@ -77,3 +84,53 @@ def init_genome_atrri_doc(kbase_collection, load_version, genome_id):
     }
 
     return doc
+
+
+def get_token(token_filename):
+    """
+    Get token if a filename is provided at home dir.
+    """
+    try:
+        home_dir = os.path.expanduser("~")
+        with open(os.path.join(home_dir, "." + token_filename), "r") as f:
+            token = f.readline().strip()
+    except FileNotFoundError as e:
+        raise e
+    return token
+
+
+def start_podman_service(uid):
+    """
+    Start podman service. Used by workspace_downloader.py script.
+    """
+    # TODO find out the right way to check if a podman service is running
```

Comment:
A simple check is waiting a little bit and doing a `poll()` to check that at least the process hasn't terminated. A better check would be reading `stdout` or `stderr` and confirming the output is as expected (if there is any)

Reply:
Similar to above.

Reply:
:+1: